src/themes/default/elements/icon.overrides

```diff
@@ -891,3 +917,13 @@ i.icon.commenting:before { content: "\f27a"; }
 i.icon.commenting.outline:before { content: "\f27b"; }
 i.icon.edge:before { content: "\f282"; }
 i.icon.ms.edge:before { content: "\f282"; }
+i.icon.wordpress.beginner:before { content: "\f297"; }
+i.icon.wordpress.forms:before { content: "\f298"; }
```

Comment:
I propose using the abbreviation forms of these two icons as base, just because are the most known way to name the brands that they represent. So, I also propose using the full forms as aliases. ¿What do you think?